### ISSUE TITLE
Composables reports table is missing some non-skippable composables

### ISSUE THREAD

**Issue by AndreasBoehm** [PatilShreyas/compose-report-to-html#9]
It looks like some composables can't be parsed from the `composables.tx` file.

### Example with a lambda that has parameters.
```
restartable scheme("[androidx.compose.ui.UiComposable]") fun MyComposable(
  unstable periods: List<DataClass>
  stable selectedPeriod: DataClass
  stable onPeriodSelected: Function2<@[ParameterName(name = 'myList')] List<DataClass>, @[ParameterName(name = 'mySingleItem')] DataClass, Unit>
  stable onCloseClicked: Function0<Unit>
)
```

### Example with missing stability information
```
restartable scheme("[androidx.compose.ui.UiComposable]") fun ComposableTwo(
  stable modifier: Modifier? = @static Companion
  data: SomeDataClass
)
```

Would be awesome if you could take a look at this.

Thanks,
Andi

**Comment by PatilShreyas** [comment]
Thanks for raising this! Will take a look.

**Comment by PatilShreyas** [comment]
This has been fixed and released in `v1.0.0-alpha03`. Thanks!
